Mathlib/CategoryTheory/Limits/Indization/IndObject.lean

```diff
@@ -185,4 +189,22 @@ theorem isIndObject_limit_comp_yoneda {J : Type u'} [Category.{v'} J] (F : J ⥤
     IsIndObject (limit (F ⋙ yoneda)) :=
   IsIndObject.map (preservesLimitIso yoneda F).hom (isIndObject_yoneda (limit F))
 
+end NonSmall
+
+section Small
+
+variable {C : Type u} [Category.{u} C]
+
+/-- Objects in a small category `C : Type u` are Ind-objects if and only if they are left-exact. -/
```

Comment:
I think it would be more precise to say
```suggestion
/-- Presheaves over a small finitely cocomplete category `C : Type u` are Ind-objects if and only if they are left-exact. -/
```